wordpress-mobile/WordPress-Android
# My Site Dashboard - Phase 2 - Create First Post Card

Parent #15200

This PR adds "create first" post card.

   ![light mode create first](https://user-images.githubusercontent.com/1405144/140905840-5c1d3535-aa2b-42a0-839d-258b2f190bc5.png)

To test:

1. Go to `My Site` -> `Me` -> `App Settings` -> `Test Feature Configuration`.
2. Turn the `MySiteDashboardPhase2FeatureConfig` feature flag `ON`.
3. Relaunch the app.
4. Note that on the `My Site` tab
    - `Create First` post card is present
    - RTL is supported

Note:
The PR includes basic styling and doesn't require a design review yet. There's a separate fine-tune UI task which will include a design review for all post cards.

## Regression Notes
1. Potential unintended areas of impact
Includes limited changes, should not impact other cards.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Added unit tests for the create first post card.

PR submission checklist:
